src/utils/prettyNum.tests.ts

```diff
@@ -0,0 +1,14 @@
+import o from 'ospec';
+import { prettyNum } from './prettyNum';
+
+o.spec('regEscape()', () => {
+	o('Escapes strings', () => {
+		o(prettyNum()).equals('0');
+		o(prettyNum(42)).equals('42');
+		o(prettyNum(123456789)).equals('123.456.789');
```

Comment:
Hvað er minnst surprising default hegðunin?

Er það íslenskt/skandínavískt formatting eða enskt?  (Ég myndi segja enskt.)

Einnig: Hvað á að gerast by default þegar ég sendi inn float?

```suggestion
		o(prettyNum(123456789)).equals('123,456,789');
		o(prettyNum(1234.56789)).equals('1,234.56789'); // <-- þetta?
		o(prettyNum(1234.56789)).equals('1,234.57'); // <-- eða þetta?
		o(prettyNum(1234567.89)).equals('1,235'); // <-- eða þetta?
```

Vænta hegðun í svona tilfellum þarf að negla í testi.

Reply:
Jafnframt: Hvað á að gerast þegar ég sendi inn weird ass broken float.

```js
o(prettyNum(1 - 0.7, 4)).equals('0.30000000000000004'); // <-- þetta?
o(prettyNum(1 - 0.7, 4)).equals('0.3000'); // <-- eða þetta?
o(prettyNum(1 - 0.7, 4)).equals('0.3'); // <-- eða þetta?
```

Þetta tengist líka "max-decimals" vs. "fixed-decimals" hegðunar pælingunum.

Reply:
Þetta er allt komið